pyproject.toml

```diff
@@ -26,7 +26,7 @@ exclude = ["**/test_*.py", "**/tests/**"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-pydantic = ">=1.10,<3.0"
+pydantic = ">=2.4,<3"
```

Comment:
Why `>=2.4` and not `>=2.0`?

Reply:
All versions before 2.4 have attached CVE. Because this change is breaking compatibility anyway, I expect using the minimal safe version is better.